drivers/windows/windows.go

```diff
@@ -421,6 +479,12 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 		endpointStruct.IPAddress = ifInfo.Address().IP
 	}
 
+	endpointStruct.DNSServerList = strings.Join(ec.DNSServers, ",")
+
+	if n.driver.name == "nat" {
+		endpointStruct.EnableInternalDNS = true
+	}
+
```

Comment:
I believe overlay networks in Windows will not be of type nat ? What will be the behavior in that case ?
If the internal DNS can't be enabled for some reason we should check for that erro and not start the resolver. Otherwise it will result in DNS requests getting dropped.